src/lib/npm-yarn-command-builder.ts

```diff
@@ -0,0 +1,18 @@
+import { IntermediateConfigCollector } from './interfaces/intermediate-config-collector';
+
+export enum BuildTools {
+  NPM,
```

Comment:
You could actually associate this to a literal string and have that used instead of an if.